yarn-project/Earthfile

```diff
@@ -100,12 +100,15 @@ anvil:
 
 end-to-end:
     FROM ubuntu:noble
-    RUN apt-get update && apt-get install -y wget gnupg \
+    # add repository for chromium
+    RUN apt-get update && apt-get install -y software-properties-common \
+        && add-apt-repository ppa:xtradeb/apps -y && apt-get update \
+        && apt-get install -y wget gnupg \
         && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
         && echo "deb [arch=$(dpkg --print-architecture)] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list \
-        && apt update && apt install curl nodejs jq google-chrome-stable netcat-openbsd -y \
+        && apt update && apt install curl chromium nodejs netcat-openbsd -y \
```

Comment:
im pretty sure this doesn't work?
i think installing chromium actually installs a wrapper that then tries to make you use snap.
which the internet seemed to have little response to.

Reply:
How is e2e-browser passing then?

Reply:
my bad. guess it does work